src/docs/tiered-storage.dox

```diff
@@ -0,0 +1,81 @@
+/*! @page tiered_storage Tiered Storage
+
+\warning The WiredTiger Tiered Storage support has not been developed, tested or tuned in
+production scenarios, and should be viewed as experimental code.
+
+@section tiered_background Background
+
+A tiered table is a single btree (possibly with multiple checkpoints) made
+up of blocks that reside in one or more files on the local file system and
+zero or more cloud objects (e.g., stored in S3). The btree spans this set of
+files and objects by encoding a block's location in the pointer that links
+blocks in the btree. A tiered btree is contrasted with a regular (non-tiered)
+btree that is contained in a single file. It is also different from an LSM
+(see @ref lsm) tree, which WiredTiger implements as multiple regular btrees,
+each in its own file.
+
+When in use, a tiered btree, like a regular btree, may have some recently used
+or modified data that resides in memory pages. This in memory representation
```

Comment:
Same comment in the other document about whether it should be `in-memory`.

Reply:
Yes, fixed the second one to be `in-memory`.